### PR TITLE
{bio}[foss/2020a] plinkQC v0.3.3

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/plinkQC/plinkQC-0.3.3-foss-2020a-R-4.0.0.eb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+exts_defaultclass = 'RPackage'
+exts_filter = ("R -q --no-save", "library(%(ext_name)s)")
+
+exts_list = [
+    ('UpSetR', '1.4.0', {
+        'checksums': ['351e5fee64204cf77fd378cf2a2c0456cc19d4d98a2fd5f3dac74b69a505f100'],
+    }),
+    (name, version, {
+        'filename': 'meyer-lab-cshl-plinkQC-%(local_git_commit)s',
```
Comment (smoors) [inline easybuild/easyconfigs/p/plinkQC/plinkQC-0.3.3-foss-2020a-R-4.0.0.eb:39]: @verdurin this line does not do anything? probably best to remove it? (also, I don't think you can do `%(local_git_commit)s`)

File: easybuild/easyconfigs/p/plinkQC/plinkQC-0.3.3-foss-2020a-R-4.0.0.eb [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        'source_tmpl': 'b83c1ec.tar.gz',
+        'source_urls': ['https://github.com/meyer-lab-cshl/plinkQC/archive/'],
+        'checksums': [None],
```
Comment (smoors) [inline easybuild/easyconfigs/p/plinkQC/plinkQC-0.3.3-foss-2020a-R-4.0.0.eb:42]: why is there no checksum?

File: easybuild/easyconfigs/p/plinkQC/plinkQC-0.3.3-foss-2020a-R-4.0.0.eb [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    }),
+]
+
+unpack_options = '--strip-components=1'
```
Comment (smoors) [inline easybuild/easyconfigs/p/plinkQC/plinkQC-0.3.3-foss-2020a-R-4.0.0.eb:46]: this line can be removed (I does not do anything for extensions)

**Review by smoors — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/p/plinkQC/plinkQC-0.3.3-foss-2020a-R-4.0.0.eb (hunk 1 above)
Comment (verdurin) [reply easybuild/easyconfigs/p/plinkQC/plinkQC-0.3.3-foss-2020a-R-4.0.0.eb:39]: @smoors yes, I originally was trying to use `git_config` which didn't seem to work well and there may be hangovers from that.

Comment (verdurin) [reply easybuild/easyconfigs/p/plinkQC/plinkQC-0.3.3-foss-2020a-R-4.0.0.eb]: Now removed.

File: easybuild/easyconfigs/p/plinkQC/plinkQC-0.3.3-foss-2020a-R-4.0.0.eb (hunk 2 above)
Comment (verdurin) [reply easybuild/easyconfigs/p/plinkQC/plinkQC-0.3.3-foss-2020a-R-4.0.0.eb]: Removed.

File: easybuild/easyconfigs/p/plinkQC/plinkQC-0.3.3-foss-2020a-R-4.0.0.eb (hunk 3 above)
Comment (verdurin) [reply easybuild/easyconfigs/p/plinkQC/plinkQC-0.3.3-foss-2020a-R-4.0.0.eb]: Also removed.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node359.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/781b8530a5dbc24aac7e04d515c04e0b for a full test report.

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @verdurin!

**Pull request merged**